core/src/components/checkbox/checkbox.ionic.vars.scss

```
@@ -1,2 +1,20 @@
+@import "../../themes/ionic.globals.ionic";
+
 // Ionic Checkbox Variables
-// --------------------------------------------------
\ No newline at end of file
+// --------------------------------------------------
+
+/// @prop - The default width and height of the checkbox
+$checkbox-ionic-size: 24px !default;
+
+/// @prop - The background color of the checkbox icon when the checkbox is unchecked
+$checkbox-ionic-icon-background-color-off: $background-color-step-450 !default;
+
+/// @prop - Border style of the checkbox icon
+$checkbox-ionic-icon-border-style: solid !default;
+
+/// @prop - Border width of the checkbox icon
+$checkbox-ionic-icon-border-width: 1px !default;
+
+/// @prop - The border radius of the checkbox icon.
+/// With a default size of 24px, the border radius is calculated as 24px / 6 = 4px
+$checkbox-ionic-border-radius: calc(var(--size) / 6) !default;
```

The original border radius value in the linked PR was incorrect. It was copied from Material Design, but in the Figma file we can see the desired border radius is:
- When the checkbox is `24px` the border radius is `4px`
- When the checkbox is small (`16px`) the border radius is `2px`.

With the current calculation, we are close enough:
- When the checkbox is `24px` the border radius is `4px`
- When the checkbox is small (`16px`) the border radius is `2.667px`

OS can target the `--border-radius` if they need to hard-set the border radius to `2px` for the small size.

This approach of basing the border radius on the size, provides a better visual appearance for when a developer customizes the size beyond the default sizes.

---

If we wanted to get it to match on both small and large we could do this:

```suggestion
$checkbox-ionic-border-radius: calc(var(--size) / 4 - 2px) !default;
```

but that's optional here.

---

I like it.